database/migrations/2024_10_30_090515_add_order_column_to_cart_items_table.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration
+{
+    public function up(): void
+    {
+        Schema::table('cart_items', function (Blueprint $table) {
+            $table->unsignedInteger('order_column')->after('vat_rate_id')->nullable()->index();
```

Comment:
I would put the order_column after total. As it is only used for sorting.
If you decide to move the order_column, adjust the ordering in the corresponding rulesets as well.